src/Components/Login.jsx

```diff
@@ -0,0 +1,55 @@
+import React, { useContext } from "react";
+import styled from "styled-components";
+import { UserContext } from "./UserContext";
+import { useGoogleLogin } from "react-google-login";
+
+export default function Login() {
+  const { setUsername, setUserEmail } = useContext(UserContext);
+  const { signIn } = useGoogleLogin({
+    clientId: "1084859424709-tk8745k1d0bnqfvlmsoa0j3uo5bkm9un.apps.googleusercontent.com",
```

Comment:
Since this is used in multiple places lets put it in a single location

Reply:
Agreed, for now moved it to "constants.js".  May want to move it to a configuration file or something like that in the future.